src/transformers/hyperparameter_search.py

```diff
@@ -0,0 +1,121 @@
+from .integrations import (
```

Comment:
Can just add a copyright here similar to all other files in the lib (potentially switching the year to 2023 if it's not)?